vital/types/metadata_traits.cxx

```diff
@@ -69,18 +70,32 @@ ::metadata_traits()
   KWIVER_VITAL_METADATA_TAGS( TABLE_ENTRY )
 
 #undef TABLE_ENTRY
+
+  // Create name table
+#define NAME_TABLE_ENTRY(TAG, NAME, TYPE, ...) \
```

Comment:
Is there a reason to not consolidate both of these into `TABLE_ENTRY`?

Reply:
Nope, just an oversight on my part.